template/README.md

```diff
@@ -404,6 +405,44 @@ Note that GitHub Pages doesn't support routers that use the HTML5 `pushState` hi
 
 Use the [Heroku Buildpack for create-react-app](https://github.com/mars/create-react-app-buildpack).
 
+### Adding Custom Environment Variables
```

Comment:
Let’s either change this to `Add` or change all other heading to remove `ing` (I think that actually might be the way to go). You can do this in a separate PR if you’d like.